src/Compilers/Metadata/CustomDebugInfoReader.cs

```diff
@@ -353,8 +353,14 @@ public static ImmutableArray<ImmutableArray<string>> GetCSharpGroupedImportStrin
                         }
 
                         int moduleInfoMethodToken = DecodeForwardToModuleRecord(record.Data);
-                        ImmutableArray<string> allModuleInfoImportStrings = reader.GetMethodByVersion(moduleInfoMethodToken, methodVersion).GetImportStrings();
-                        ArrayBuilder<string> externAliasBuilder = ArrayBuilder<string>.GetInstance();
+
+                        ImmutableArray<string> allModuleInfoImportStrings = getMethodImportStrings(moduleInfoMethodToken, arg);
+                        if (allModuleInfoImportStrings.IsDefault)
+                        {
+                            throw new InvalidOperationException(string.Format("Forwarded to non-existing method {0}", FormatMethodToken(moduleInfoMethodToken)));
```

Comment:
Previous code throw NullReferenceException. So I think this is better because we do handle InvalidOperationException in the EE